modules/oneVideoBidAdapter.js

```diff
@@ -165,14 +165,17 @@ function getRequestData(bid, consentData, bidRequest) {
   let loc = bidRequest.refererInfo.referer;
   let page = (bid.params.site && bid.params.site.page) ? (bid.params.site.page) : (loc.href);
   let ref = (bid.params.site && bid.params.site.referrer) ? bid.params.site.referrer : bidRequest.refererInfo.referer;
+  let getFloorRequestObject = {
+    currency: bid.params.cur || 'USD',
+    mediaType: 'video',
+    size: '\*'
```

Comment:
In the docs, in one example the passed just `'*'` and in another example they used `'\*'`, do you think there will be any difference in calculation if we used `escape sequence` for `*`.

Reply:
Also just a reminder, is there anything that needs to be updated in `adapter md` file.

Apart from this everything looks fine @adam-browning.

Thank you  😊

Reply:
I didn't see any differences in the results after testing both.
I am removing the escaped character backslash.